scripts/scil_fodf_memsmt.py

```diff
@@ -160,17 +160,19 @@ def main():
 
     affine = extract_affine(args.in_dwis)
 
-    tol = args.tolerance
-
     wm_frf = np.loadtxt(args.in_wm_frf)
     gm_frf = np.loadtxt(args.in_gm_frf)
     csf_frf = np.loadtxt(args.in_csf_frf)
 
+    # Note. This script does not currently allow using a separate b0_threshold
```

Comment:
Same here, I think we should add a `check_b0_threshold` using `args.tolerance`.

Reply:
Ok, but if I do, with the --skip_b0_validation, the args.tolerance value could eventually be changed.

Reply:
Not sure I understand what you mean here